temporalcli/commandsmd/commands.md

```diff
@@ -1311,6 +1311,9 @@ temporal server start-dev \
 * `--ui-ip` (string) -
   IP address bound to the WebUI.
   Default is same as '--ip' value.
+* `--public-path` (string) -
+  The public base path for the Web UI.
+  Default is `/`
```

Comment:
```suggestion
  Default is `/`.
```

Agree with Chad's request to add the `ui-` prefix to this option; we'd like to be consistent with the other UI options.

Otherwise this looks good to me!